beater/config.go

```diff
@@ -258,6 +259,7 @@ func defaultConfig(beatVersion string) *Config {
 						filepath.Join("ingest", "pipeline", "definition.json")),
 				}},
 		},
-		Mode: ModeProduction,
+		Mode:   ModeProduction,
+		Kibana: nil,
```

Comment:
Why are you setting a nil default?

Reply:
Because a default of `common.Config()` would imply enabled=true.
That means that after upgrading, users would be welcome with a new requirement (Kibana) or an error in the logs.

Reply:
Ok. I'd be more explicit about it by either setting ```common.NewConfigFrom(`enabled: false`)``` or adding a dedicated test to the `config_test.go` checking that `Kibana.Enabled() == false`. Otherwise it could easily happen that this gets removed in the future without anything failing.